dbms/tests/queries/0_stateless/00678_murmurhash.reference

```diff
@@ -1,26 +1,25 @@
-623211862
-3533626746
-2388617433
-2708309598
+3012058918
```

Comment:
Don't understand, why it was changed...

Reply:
It's just changed to `intHash32` or `intHash64` as it was for other non-crypto hash functions.
I don't think it is expected behaviour.